environment.yaml

```diff
@@ -12,3 +12,7 @@ dependencies:
   - openssl
   - ripgrep
   - poetry=1.1
+  - mykrobe=0.10
```

Comment:
0.11 was just released, but I am just trying to update the conda recipe to have the correct dependencies

Reply:
ok, bioconda release of v0.11.0 should be available now

Reply:
Done in https://github.com/mbhall88/tbpore/pull/8/commits/10b410e9f9e6f865507f1c9d70e17d4418602461